src/Components/SignPages/Check.jsx

```diff
@@ -1,9 +0,0 @@
-import React from 'react'
-
-function Check() {
```

Comment:
We can remove this component and use div directly where we need it. This component is unnecessary.